R/dependencies.R

```diff
@@ -80,7 +80,7 @@ snapshotDependencies <- function(appDir, implicit_dependencies=c()) {
   } else {
```

Comment:
`available.packages` is also called in line 79 for `biocRepos`, do we need to filter on `R_version` there too?

Reply:
R version filtering feels appropriate any time we might encounter repositories that do not contain the entire release history for some package. That probably includes the BioC repositories.

Reply:
Done.